recipes/SSp_Detector/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "SSp_Detector" %}
+{% set version = "2023.10.25rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/SSp_Detector-{{ version }}.tar.gz
+  sha256: 019b3b7bb498732881ebda44dadae3f2ec0f3493da23d9a7d6b6ebcb6eba48b2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.9.0
+  run:
+    - python >=3.9.0
+    - mne >=1.5.1
+    - numpy >=1.24.3
+    - outdated
+    - pandas >=2.0.0
+    - scipy >=1.11
+
+test:
+  imports:
+    - SSp_Detector
+
+about:
+  home: https://github.com/sjg2203/SSp_Detector
+  summary: Sleep spindles detector
+  license: Apache-2.0
+  license_file: LICENSE
+  doc_url: https://github.com/sjg2203/SSp_Detector/blob/main/README.md
+  dev_url: https://github.com/sjg2203/SSp_Detector
+
+extra:
+  recipe-maintainers:
+    - sjg2203
```

Comment:
You are still removing the pypromice recipe. Let me rebase this for you.

Reply:
Apologies, the example I saw was removing pypromice. Thanks for the help